extensions/amp-story-360/0.1/test/validator-amp-story-360.out

```diff
@@ -1,4 +1,4 @@
-PASS
+FAIL
```

Comment:
Should all examples in this file be valid? If not could we update the comments for each instance in why they are not valid? I'm going to guess that this new example should be valid, so we'll need to add `amp-video` extension and the `poster` attribute.

Reply:
+1 I think it should be valid